website/dbt-versions.js

```diff
@@ -42,6 +42,10 @@ exports.versions = [
  * @property {string} lastVersion The last version the page is visible in the sidebar
  */
 exports.versionedPages = [
+  {
+    page: "docs/build/incremental-microbatch",
+    lastVersion: "1.9",
```

Comment:
i think its meant to show for 1.9 and higher right? it wasn't showing when i was on versionless, however it was showing for 1.8 and lower
```suggestion
    firstVersion: "1.9",
```

Reply:
you're totally right!